src/test/java/com/segment/analytics/android/integrations/amplitude/AmplitudeTest.java

```diff
@@ -84,15 +84,21 @@ public void initialize() {
         new ValueMap().putValue("apiKey", "foo")
             .putValue("trackAllPages", true)
             .putValue("trackCategorizedPages", false)
-            .putValue("trackNamedPages", true));
+            .putValue("trackNamedPages", true)
+            .putValue("enableLocationListening", true)
+            .putValue("useAdvertisingIdForDeviceId", true));
 
     assertThat(integration.trackAllPages).isTrue();
     assertThat(integration.trackCategorizedPages).isFalse();
     assertThat(integration.trackNamedPages).isTrue();
+    assertThat(integration.enableLocationListening).isTrue();
```

Comment:
Can remove these two new lines and asserts. The verify test is sufficient for this case.